### ISSUE TITLE
Comment formuler qu'une entreprise ne peut pas faire de détournement commercial 

### ISSUE THREAD

**Issue by BodySplash** [ordre-des-developpeurs/odd-constitution#9]
Une entreprise ne peut pas se prévaloir de faire partie de l'ordre 
C'est une démarche personnelle.

**Comment by BodySplash** [comment]
Tu ne peux mentionner avoir des membres de l'ordre sans les citer. La citation doit être autorisée par le membre.

**Comment by BodySplash** [comment]
Ne pas pouvoir être inquiété de ne pas donner cette autorisation

**Comment by kryzaal** [comment]
Bien sûr, c'est ce qui différentie un ordre d'une fédération. Après chacun est libre de se réclamer de telle entreprise, tout comme telle entreprise peut réclamer avoit n développeurs dans l'ordre.

Je verrais plutôt la règle comme un inclusion plutôt que comme des exclusions :
"L'ordre est constitué uniquement de personnes physiques exerçant la profession de développeur"

**Comment by BodySplash** [comment]
Justement, la discussion de samedi était de ne pas permettre aux entreprises d'afficher un «nous avons tant de développeurs dans l'ordre» Pour éviter ainsi l'effet qu'il y a eu avec le Software craftsmanship «oui oui on adhère», alors que dans les faits, rien du tout. 
Cette obligation d'y adjoindre les noms, nous semblait une bonne idée pour éviter les mensonges, et permettre aux clients de valider avec quelles personnes il va travailler.

**Comment by abelards** [comment]
Je crois que tu es très clair avec les valeurs, déjà.

Je trouve très sain de dire "je crois aux valeurs du Agile Manifesto et du SWCraftsmanship",
je trouverais très sain de dire "notre lead dev souhaite créér un environnement partageant les valeurs de l'Ordre des Devs", tout comme je peux dire "notre boîte a X/12 au Joel Test".

Mais pas "ma boîte adhère à X ou Y", qui elle n'a aucun sens [1], et j'irai jusqu'à dire "je suis de l'Ordre des développeurs" n'a pas beaucoup de sens pratique non plus.
C'est le fameux "annoncer des valeurs et vivre ses valeurs, c'est pas la même chose".
Mais ça, c'est vrai aussi des boîtes comme des gens.

[1] en tant que "vieux dev" je sais très bien lire "un chef a donné des sous ou signé un truc à telle asso, ça n'impacte pas forcément sa manière dont les équipes sont managées en vrai"
